src/AppBundle/Entity/Repository/ClientRepository.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace AppBundle\Entity\Repository;
+
+use AppBundle\Entity\Client;
+use Doctrine\ORM\EntityRepository;
+use AppBundle\Entity\User;
+
+/**
+ * ClientRepository.
+ *
+ * This class was generated by the Doctrine ORM. Add your own custom
+ * repository methods below.
+ */
+class ClientRepository extends EntityRepository
+{
+
+    /**
+     * Search Clients
+     *
+     * @param string $query Search query
+     * @param string $orderBy field to order by
+     * @param string $sortOrder order of field order ASC|DESC
+     * @param int $limit number of results to return
+     * @param string $offset
+     *
+     * @return Client[]|array
+     */
+    public function searchClients($query = '', $orderBy = 'lastname', $sortOrder = 'ASC', $limit = 100, $offset = 'id')
```

Comment:
default param "id", should be 0

Reply:
since it's called with all the params, I don't think it's bad to remove default params. we'll shift the responsability to the caller, and simplify things

Reply:
Yes, copied from somewhere else. Ill change. Need to force the limit to be max 100 too